src/RetrySettings.php

```diff
@@ -405,6 +461,9 @@ public function with(array $settings)
             'noRetriesRpcTimeoutMillis' => $this->getNoRetriesRpcTimeoutMillis(),
             'maxRetries' => $this->getMaxRetries(),
             'retryFunction' => $this->getRetryFunction(),
+            'retryDelayFunction' => $this->getRetryDelayFunction(),
```

Comment:
I thought we had the general agreement that we want to remove the delayFunction and only give option to add floats?

Reply:
What if the delay needs to be retrieved from the server exception? Thats a usecase for the Spanner